lib/Bio/EnsEMBL/DataCheck/Checks/CheckOrthologyQCThresholds.pm

```diff
@@ -39,18 +40,34 @@ use constant {
 sub tests {
   my ($self) = @_;
   my $dbc = $self->dba->dbc;
-  my @thresholds = qw( goc_quality_threshold wga_quality_threshold );
-
-  foreach my $threshold ( @thresholds ) {
-    my $desc = "There are some $threshold in method_link_species_set_attr";
-    my $sql = qq/
-      SELECT COUNT(*) 
-        FROM method_link_species_set_attr 
-      WHERE $threshold IS NOT NULL
-    /;
-    is_rows_nonzero( $dbc, $sql, $desc );
+
+  my $goc_sql = qq/
+    SELECT COUNT(*)
+      FROM method_link_species_set_attr
+    WHERE goc_quality_threshold IS NOT NULL
+  /;
+  my $has_goc_scores = sql_count($self->dba, 'SELECT 1 FROM homology WHERE goc_score IS NOT NULL LIMIT 1');
+  if ($has_goc_scores) {
+    my $desc = "There are some goc_quality_threshold in method_link_species_set_attr";
+    is_rows_nonzero( $dbc, $goc_sql, $desc );
+  } else {
+    my $desc = "There are no goc_quality_threshold in method_link_species_set_attr";
+    is_rows_zero( $dbc, $goc_sql, $desc );
   }
 
+  my $wga_sql = qq/
+    SELECT COUNT(*)
+      FROM method_link_species_set_attr
+    WHERE wga_quality_threshold IS NOT NULL
+  /;
+  my $has_wga_scores = sql_count($self->dba, 'SELECT 1 FROM homology WHERE wga_coverage IS NOT NULL LIMIT 1');
+  if ($has_wga_scores) {
+    my $desc = "There are some wga_quality_threshold in method_link_species_set_attr";
+    is_rows_nonzero( $dbc, $wga_sql, $desc );
+  } else {
+    my $desc = "There are no wga_quality_threshold in method_link_species_set_attr";
```

Comment:
To stick with consistency, the test descriptions should be passing descriptions only. There shouldn't be a need to include an else with the opposing statement, @james-monkeyshines mentioned in one of my PRs that you can include a SKIP block, perhaps this here would be a good place to implement it?

Reply:
The test in the _else_ block is actually different. On one side it tests that there are rows, on the other side it tests that there no rows. I could combine both tests into something like:
- get the number of rows
- depending on the `has*` boolean, set the comparison operator to `==` or `>`
- use `cmp_ok` with a description like "There is the expected number of rows with wga_quality_threshold"

Reply:
I like that idea with the comparison operator, that way it is just a single positive description for both